<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数值分析</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>响应列表</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <row>
     <property name="text">
      <string>传感器1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>传感器15</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>响应时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>峰值时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>峰值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>最大正斜率</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>最大负斜率</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>恢复时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>响应峰面积</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>恢复峰面积</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="openButton">
    <property name="text">
     <string>Open Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analysisButton">
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
